x-pack/platform/plugins/private/upgrade_assistant/__jest__/client_integration/es_deprecations/reindex_deprecation_flyout.test.ts

```diff
@@ -91,7 +79,7 @@ describe('Reindex deprecation flyout', () => {
 
     const { actions, exists } = testBed;
 
-    await actions.table.clickDeprecationRowAt('reindex', 0);
+    await actions.table.clickDeprecationRowAt('reindex', 0, 'reindex');
```

Comment:
sending the reindex string twice looks super weird 😬 I guess its a side effect of having the function take literal parameters instead of a config object